### PR TITLE
KOGITO-204: Fix locators for editor components in vscode

### PR DESCRIPTION
**Pull request by domhanak**
waiting for #382

### CODE REVIEW

File: packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/BpmnEditorTestHelper.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class BpmnEditorTestHelper {
         const properties = await this.getDiagramProperties();
         await assertWebElementIsDisplayedEnabled(properties)
         await properties.click();
-        const expandedPropertiesPanel = await this.webview.findWebElement(By.className('qe-docks-bar-expanded-E'))
+        const expandedPropertiesPanel = await this.webview.findWebElement(By.className('docks-bar-expanded-E'))
```
Comment (jomarko) [inline packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/BpmnEditorTestHelper.ts]: In whole PR `qe-docsk-bar-expanded-[XYZ]` should be `expanded-docks-bar-[XYZ]`

**Review by jomarko — CHANGES_REQUESTED** [review]

File: packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/BpmnEditorTestHelper.ts (hunk 1 above)
Comment (domhanak) [reply packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/BpmnEditorTestHelper.ts]: Ok, thanks. Will update

File: packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/CommonLocators.ts [hunk 2]
```diff
@@ -39,4 +41,11 @@ export const spanComponentWithText = (text: string): string => {
  */
 export const h3ComponentWithText = (text: string): string => {
     return `//h3[text() = \'${text}\']`
+}
+
+/**
+ * Creates a classname locator for 'expanded-docks-bar-E'
+ */
+export const expandedDocksBarE = (): By => {
```
Comment (jomarko) [inline packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/CommonLocators.ts:49]: Optional proposal, me personally would use `expandedDocksBarEeast` as the name. Let me know what do you think. Really not necessary. Just point for discussion.

Comment (domhanak) [reply packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/CommonLocators.ts:49]: Sounds good to me, I was thinking something similar, I will change it to `expandedDocksBarEast` for W we can add `expandedDocksBarWest` when needed

File: packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/DmnEditorTestHelper.ts [hunk 3]
```diff
@@ -65,7 +65,7 @@ export default class DmnEditorTestHelper {
      * @returns Promise<WebElement> promise that resolves to DMN decision navigator element. 
      */
     public getDecisionNavigator = async (): Promise<WebElement> => {
-        this.decisionNavigator = await this.webview.findWebElement(By.className('qe-docks-item-E-org.kie.dmn.decision.navigator'));
+        this.decisionNavigator = await this.webview.findWebElement(By.className('docks-item-E-org.kie.dmn.decision.navigator'));
```
Comment (domhanak) [inline packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/DmnEditorTestHelper.ts:68]: @jomarko what about unification of navigator/explorer to `docks-item-E-DiagramExplorer` for every editor? Similar to properties class. The idea is that decision navigator and diagram explorer are the same things, not sure if it is possible though.

Comment (jomarko) [reply packages/vscode-extension-pack-kogito-kie-editors/src/ui-test/helpers/DmnEditorTestHelper.ts:68]: @domhanak not sure I get your point. `docks-item-E-org.kie.dmn.decision.navigator` and `docks-item-E-DiagramExplorer` are both placed in the east of the screen but both has different content.

**Comment by tiagobento** [comment]
#382 is merged. Is this one good for review?

**Comment by jomarko** [comment]
@tiagobento yes, I think so.

**Review by jomarko — APPROVED** [review]

**Comment by tiagobento** [comment]
@domhanak Checks are failing, can you please take a look?

**Comment by jomarko** [comment]
Closing in favor of #395 

@domhanak thank you for help with locators updates

**Pull request closed without merging**